src/dash-playlist-loader.js

```diff
@@ -194,7 +194,7 @@ export const filterChangedSidxMappings = (masterXml, srcUrl, clientOffset, oldSi
 export const requestSidx_ = (loader, sidxRange, playlist, xhr, options, finishProcessingFn) => {
   const sidxInfo = {
     // resolve the segment URL relative to the playlist
-    uri: resolveManifestRedirect(options.handleManifestRedirects, sidxRange.resolvedUri),
+    uri: resolveManifestRedirect(options.handleManifestRedirects, sidxRange.resolvedUri, xhr),
```

Comment:
I believe the `xhr` here isn't actually the request but just a reference to the `xhr` method as defined in `xhr.js`.